conan/tools/cmake/cmakedeps/templates/macros.py

```diff
@@ -28,7 +28,7 @@ def template(self):
         return textwrap.dedent("""
         function(conan_message MESSAGE_TYPE MESSAGE_CONTENT)
             if(NOT CONAN_CMAKE_SILENT_OUTPUT)
-                message(${MESSAGE_TYPE} ${MESSAGE_CONTENT})
+                message(${MESSAGE_TYPE} "${MESSAGE_CONTENT}")
```

Comment:
Without quotes, it removes the `;` from lists... which can be very confusing when you are focused on an issue related to a list of paths 😄